core/src/net/sf/openrocket/rocketcomponent/FinSet.java

```diff
@@ -264,16 +264,16 @@ public double getTabHeight() {
 	 * Note this function also does bounds checking, and will not set
 	 * a tab height that passes through it's parent's midpoint.
 	 *  
-	 * @param newHeightRequest how deep the fin tab should project
+	 * @param newTabHeightRequest how deep the fin tab should project
 	 * from the fin root, at the reference point
 	 * 
 	 */
-	public void setTabHeight(final double heightRequest) {
-		if (MathUtil.equals(this.tabHeight, MathUtil.max(heightRequest, 0))){
+	public void setTabHeight(final double newTabHeightRequest) {
```

Comment:
nit: Since you're changing the variable name anyways, does the Request part of the name really make any sense? I think its just as clear to set it as newTabHeight.

Reply:
Yea, good point.  remove the 'request' suffix in this method.